target-platforms/target-platform-latest/target-platform-latest.target

```diff
@@ -13,7 +13,7 @@
 			<unit id="org.eclipse.lsp4j.jsonrpc" version="0.0.0"/>
 			<unit id="org.eclipse.lsp4j.debug" version="0.0.0"/>
 			<unit id="org.eclipse.lsp4j.jsonrpc.debug" version="0.0.0"/>
-			<repository location="https://download.eclipse.org/lsp4j/updates/releases/0.18.0/"/>
+			<repository location="https://download.eclipse.org/lsp4j/updates/releases/0.19.0/"/>
```

Comment:
this file (some lines above) still used the old orbit/guava with the signing issue